libreyolo/models/rfdetr/transformer.py

```
@@ -776,6 +1126,101 @@ def forward_post(
         tgt = self.norm3(tgt)
+
+        # --- GroupPose keypoint additions (ported from RF-DETR v1.8.0). ---
+        if self.enable_keypoint_processing:
+            if keypoint_tgt is None or keypoint_pos is None:
+                raise ValueError("Keypoint processing is enabled but keypoint_tgt/keypoint_pos missing")
+
+            tgt_for_kp = self.inst_in_proj(tgt)
+            tgt_for_kp_pos = self.inst_pos_in_proj(query_pos)
+
+            # ========== Begin of Keypoint-Instance Self-Attention =============
+            _, n_queries, num_kp, kp_dim = keypoint_tgt.shape
```

Ruff (0.15.17)

[warning] Unpacked variable `n_queries` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)